src/mountebank.js

```diff
@@ -148,6 +148,54 @@ function create (options) {
         });
     });
 
+    function portStored (saveImpostersFile, saveImpostersFileFlag) {
+        var fs = require('fs');
+        var portStore = [];
+        var ImposterDir = './Repository_Template';
+        if (fs.existsSync(ImposterDir + '/' + saveImpostersFile) === true) {
```

Comment:
I'm having trouble understanding this block. It looks to be failing if the savefile already exists, no? It looks to be doing it by parsing the file and adding each imposter's port to an array, but if we add ANYTHING to the array, it will fail with an error message around port usage, which I don't understand since we're not checking ports. Can you explain the error condition to me?

FWIW, I don't think we even bother with the check. I think we simply overwrite the file if it exists. That will likely change the order of loading the file loading in bin/mb, because there's a race condition if someone starts by 'mb --configfile mb.json --savefile mb.json' -- we'd still want to load the configfile before overwriting it.

(Upon further review, it appears to be copy/pasted from other parts of the code where the check is more legit. As I understand it, this is not needed here).

Reply:
This was done as a warning (not error) so the user is prompted of the ports used by saved imposters in the previous session.  
```
warn: [mb:2525] These ports are already used 7667 use unique ports for imposters. Check ./Repository_Template/mb.json for stored imposter collections 
```
However, if we want to overwrite every time this is indeed unnecessary.

Reply:
In addition, overwriting the existing file will not make it persistent and amount to loss of older session data.